suse-x64/Dockerfile

```diff
@@ -165,25 +168,39 @@ RUN curl -sL -O "https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.t
 # Use our updated binutils version
 ENV PATH="/usr/local/binutils/bin:$PATH"
 
+# xz needs gettext 0.19.6 or newer
+RUN curl -LO https://ftp.gnu.org/gnu/gettext/gettext-${GETTEXT_VERSION}.tar.xz \
+    && echo "${GETTEXT_SHA256}  gettext-${GETTEXT_VERSION}.tar.xz" | sha256sum --check \
+    && tar xf "gettext-${GETTEXT_VERSION}.tar.xz" \
+    && cd "gettext-${GETTEXT_VERSION}" \
+    && ./configure --prefix=/usr && make -j$(nproc) && make install \
+    && cd / \
+    && rm -rf "gettext-${GETTEXT_VERSION}"
```

Comment:
nit - I think you do not remove the `tar.gz` file, just the folder.

Reply:
Good point

Reply:
Done, and replicated to the debian dockerfile which I copy pasted from